backend/tests/troops.test.js

```diff
@@ -0,0 +1,21 @@
+const request = require('supertest');
+const app = require('../App');
+
+describe('Troops', () => {
+  describe('GET /kingdom/troops', () => {
+    it('should return 200 when getting all troops', (done) => {
+      request(app)
+        .get('/kingdom/troops')
+        .expect('Content-Type', /json/)
+        .expect(200, done);
+    });
+  });
+
+  describe('GET /kingdom/non-exists-endpoint', () => {
```

Comment:
we still need another test for troops in the situation of not getting 200.

Reply:
you mean I should test like this
`/kingdom/troops/non-exist`